near/res/testnet_config.json

```diff
@@ -0,0 +1,208 @@
+            "execution": 37538150
+          }
+        },
+        "delete_key_cost": {
+          "send_sir": 0,
+          "send_not_sir": 0,
+          "execution": 0
+        },
+        "delete_account_cost": {
+          "send_sir": 454830000000,
+          "send_not_sir": 454830000000,
+          "execution": 454830000000
+        }
+      },
+      "storage_usage_config": {
+        "account_cost": 100,
+        "data_record_cost": 40,
+        "key_cost_per_byte": 1,
+        "value_cost_per_byte": 1,
+        "code_cost_per_byte": 1
+      },
+      "burnt_gas_reward": {
+        "numerator": 3,
+        "denominator": 10
+      }
+    },
+    "wasm_config": {
+      "ext_costs": {
+        "base": 126224222,
+        "read_memory_base": 1629369577,
+        "read_memory_byte": 123816,
+        "write_memory_base": 76445225,
+        "write_memory_byte": 809907,
+        "read_register_base": 639340699,
+        "read_register_byte": 63637,
+        "write_register_base": 0,
+        "write_register_byte": 0,
+        "utf8_decoding_base": 0,
+        "utf8_decoding_byte": 591904,
+        "utf16_decoding_base": 0,
+        "utf16_decoding_byte": 9095538,
+        "sha256_base": 710092630,
+        "sha256_byte": 5536829,
+        "keccak256_base": 710092630,
+        "keccak256_byte": 5536829,
+        "keccak512_base": 1420185260,
+        "keccak512_byte": 11073658,
+        "log_base": 0,
+        "log_byte": 0,
+        "storage_write_base": 21058769282,
+        "storage_write_key_byte": 23447086,
+        "storage_write_value_byte": 9437547,
+        "storage_write_evicted_byte": 0,
+        "storage_read_base": 19352220621,
+        "storage_read_key_byte": 4792496,
+        "storage_read_value_byte": 139743,
+        "storage_remove_base": 109578968621,
+        "storage_remove_key_byte": 9512022,
+        "storage_remove_ret_value_byte": 0,
+        "storage_has_key_base": 20019912030,
+        "storage_has_key_byte": 4647597,
+        "storage_iter_create_prefix_base": 28443562030,
+        "storage_iter_create_prefix_byte": 442354,
+        "storage_iter_create_range_base": 25804628282,
+        "storage_iter_create_from_byte": 429608,
+        "storage_iter_create_to_byte": 1302886,
+        "storage_iter_next_base": 24213271567,
+        "storage_iter_next_key_byte": 0,
+        "storage_iter_next_value_byte": 1343211668,
+        "touching_trie_node": 1,
+        "promise_and_base": 0,
+        "promise_and_per_promise": 672136,
+        "promise_return": 34854215
+      },
+      "grow_mem_cost": 1,
+      "regular_op_cost": 3856371,
+      "max_gas_burnt": 200000000000000,
+      "max_stack_height": 65536,
+      "initial_memory_pages": 17,
+      "max_memory_pages": 32,
+      "registers_memory_limit": 1073741824,
+      "max_register_size": 104857600,
+      "max_number_registers": 100,
+      "max_number_logs": 100,
+      "max_log_len": 500
+    },
+    "account_length_baseline_cost_per_block": "6561"
```

Comment:
This is wrong. Where does it came from? It's different from the current testnet.json from staging.

Reply:
From node1:
![image](https://user-images.githubusercontent.com/13259400/74761519-05a81c80-5231-11ea-8d25-0a453c5cf99d.png)
Also this is going to merge to master